### PR TITLE
Template  display.tpl must extend page.tpl

### PR DESCRIPTION
**Pull request by fabienVernieres**
Currently the template displays raw non-nested content.

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x / 8.x
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

File: modules/creation/displaying-content-in-front-office.md [hunk 1]
```diff
@@ -309,7 +309,10 @@ class mymoduledisplayModuleFrontController extends ModuleFrontController
 - *display.tpl*
 
 ```
```
Comment (kpodemski) [inline modules/creation/displaying-content-in-front-office.md]:
```suggestion
```smarty
```

File: modules/creation/displaying-content-in-front-office.md [hunk 2]
```diff
@@ -391,7 +394,10 @@ in our TPL file.
 - *mymodule.tpl*
 
 ```
```
Comment (kpodemski) [inline modules/creation/displaying-content-in-front-office.md]:
```suggestion
```smarty
```

**Comment by fabienVernieres** [comment]
Done @kpodemski

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @fabienVernieres :)

**Pull request merged**